src/storage/data.rs

```diff
@@ -27,6 +117,54 @@ where
 }
 
 /// A storage with read and write access.
+///
+/// In addition to `ReadStorage`, a storage with mutable access allows to:
```

Comment:
Additionally to what `ReadStorage` can do a storage with mutable access allows:

Reply:
My sentence doesn't make any sense at all, thanks for the suggestion.